packages/@core.keystone/fields/Json/utils/cleaner.js

```diff
@@ -0,0 +1,20 @@
+const { isArray, isObject, omit } = require('lodash')
```

Comment:
Its a good practice to write a `.spec` with tests if you write utils

Reply:
tests added 👍